main.tf

```diff
@@ -25,13 +25,10 @@ resource "azurerm_cdn_endpoint" "cdn_endpoint" {
   profile_name                  = local.cdn_endpoint[each.key].profile_name
   is_http_allowed               = local.cdn_endpoint[each.key].is_http_allowed
   is_https_allowed              = local.cdn_endpoint[each.key].is_https_allowed
-  content_types_to_compress     = local.cdn_endpoint[each.key].content_types_to_compress
   is_compression_enabled        = local.cdn_endpoint[each.key].is_compression_enabled
   querystring_caching_behaviour = local.cdn_endpoint[each.key].querystring_caching_behaviour
   optimization_type             = local.cdn_endpoint[each.key].optimization_type
   origin_host_header            = local.cdn_endpoint[each.key].origin_host_header
-  origin_path                   = local.cdn_endpoint[each.key].origin_path
```

Comment:
why removing  [origin_path] (https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_endpoint#origin_path-1) it still exists int the latest version auf azurerm 4